plugins/techdocs/src/home/components/TechDocsCustomHome.tsx

```diff
@@ -153,7 +172,7 @@ export const TechDocsCustomHome = (props: TechDocsCustomHomeProps) => {
     error,
   } = useAsync(async () => {
     const response = await catalogApi.getEntities({
-      filter: {
+      filter: filter || {
```

Comment:
This should probably be merging these together. Not having the TechDocs annotation essentially means you have the same functionality as the Catalog index page.